app/components/forms/wizard/basic-details-step.js

```diff
@@ -404,7 +373,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
       }
     };
     // Merging the predetermined rules with the rules for social links.
-    validationRules.fields = merge(validationRules.fields, this.socialLinksValidationRules);
+   // validationRules.fields = merge(validationRules.fields, this.socialLinksValidationRules);
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces but found 3.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6089057)